opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java

```diff
@@ -34,127 +33,116 @@
 @DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 class OpenSearchAggregationResponseParserTest {
 
-  /**
-   * SELECT MAX(age) as max FROM accounts.
-   */
+  /** SELECT MAX(age) as max FROM accounts. */
   @Test
   void no_bucket_one_metric_should_pass() {
-    String response = "{\n"
-        + "  \"max#max\": {\n"
-        + "    \"value\": 40\n"
-        + "  }\n"
-        + "}";
-    NoBucketAggregationParser parser = new NoBucketAggregationParser(
-        new SingleValueParser("max")
-    );
+    String response = "{\n" + "  \"max#max\": {\n" + "    \"value\": 40\n" + "  }\n" + "}";
```

Comment:
Can you keep it multiline?
```suggestion
    String response =
        "{\n"
            + "  \"max#max\": {\n"
            + "    \"value\": 40\n"
            + "  }\n"
            + "}";
```
If not, simplify the string. I think we can omit whitespaces.
```suggestion
    String response = "{ \"max#max\": { \"value\": 40\ } }";
```